### PR TITLE
[SPARK-28806] [DOCS] [FOLLOW-UP] Remove unneeded HTML from the MD file

### PR DESCRIPTION
**Pull request by gatorsmile**
### What changes were proposed in this pull request?
This PR is to clean up the markdown file in SHOW COLUMNS page.

- remove the unneeded embedded inline HTML markup by using the basic markdown syntax.
- use the ``` sql for highlighting the SQL syntax.

### Why are the changes needed?
Make the doc cleaner and easily editable by MD editors.

### Does this PR introduce _any_ user-facing change?
NO

### How was this patch tested?
**Before**

Screen Shot 2020-04-29 at 5 20 11 PM

**After**

Screen Shot 2020-04-29 at 6 03 50 PM

### CODE REVIEW

File: docs/sql-ref-syntax-aux-show-columns.md [hunk 1]
```diff
@@ -25,41 +25,34 @@ Return the list of columns in a table. If the table does not exist, an exception
...
     is specified. Please note that when this parameter is specified then table
-    name should not be qualified with a different database name. <br><br>
-    <b>Syntax:</b>
-      <code>
-        { IN | FROM } database_name
-      </code><br><br>
-    <b>Note:</b>
-    Keywords <code>IN</code> and <code>FROM</code> are interchangeable.
-  </dd>
-</dl>
```
Comment (gatorsmile) [inline docs/sql-ref-syntax-aux-show-columns.md:58]: Let us avoid using HTML if possible.

File: docs/sql-ref-syntax-aux-show-columns.md [hunk 2]
```diff
@@ -25,41 +25,34 @@ Return the list of columns in a table. If the table does not exist, an exception
...
+    name should not be qualified with a different database name. 
+
+    **Syntax:** `{ IN | FROM } database_name`
+
+    **Note:** Keywords `IN` and `FROM` are interchangeable.
 
 ### Examples
 
-{% highlight sql %}
```
Comment (gatorsmile) [inline docs/sql-ref-syntax-aux-show-columns.md:62]: We can directly use ```sql

File: docs/sql-ref-syntax-aux-show-columns.md [hunk 3]
```diff
@@ -96,9 +89,9 @@ SHOW COLUMNS IN customer IN salesdb;
   |     name|
   |cust_addr|
   +---------+
-{% endhighlight %}
+```
 
 ### Related Statements
 
- * [DESCRIBE TABLE](sql-ref-syntax-aux-describe-table.html)
```
Comment (gatorsmile) [inline docs/sql-ref-syntax-aux-show-columns.md:103]: unneeded space.

**Comment by gatorsmile** [comment]
cc @huaxingao Could you help fix all the other pages like this example?

File: docs/sql-ref-syntax-aux-show-columns.md [hunk 4]
```diff
@@ -25,41 +25,34 @@ Return the list of columns in a table. If the table does not exist, an exception
...
+    with a database name.
+
+    **Syntax:** `{ IN | FROM } [ database_name . ] table_name`
+
+    **Note:** Keywords `IN` and `FROM` are interchangeable.
+
+* **database**
+
     Specifies an optional database name. The table is resolved from this database when it
-    is specified. Please note that when this parameter is specified then table
```
Comment (gatorsmile) [inline docs/sql-ref-syntax-aux-show-columns.md:49]: Nit: removed `Please note that `

**Comment by gatorsmile** [comment]
Also cc @dilipbiswal

**Comment by dilipbiswal** [comment]
LGTM

**Review by srowen — APPROVED** [review]

**Comment by gatorsmile** [comment]
Thanks! Merged to master/3.0

**Pull request closed without merging**